[2025-05-21T16:21:35.025+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-21T16:21:35.151+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2025-04-22T00:00:00+00:00 [queued]>
[2025-05-21T16:21:35.202+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2025-04-22T00:00:00+00:00 [queued]>
[2025-05-21T16:21:35.203+0000] {taskinstance.py:2884} INFO - Starting attempt 1 of 6
[2025-05-21T16:21:35.257+0000] {taskinstance.py:2907} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2025-04-22 00:00:00+00:00
[2025-05-21T16:21:35.268+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=11826) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-21T16:21:35.281+0000] {standard_task_runner.py:72} INFO - Started process 11843 to run task
[2025-05-21T16:21:35.294+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name', 'scheduled__2025-04-22T00:00:00+00:00', '--job-id', '3829', '--raw', '--subdir', 'DAGS_FOLDER/dar_with_taskflow_api.py', '--cfg-path', '/tmp/tmp8dqax1qb']
[2025-05-21T16:21:35.296+0000] {standard_task_runner.py:105} INFO - Job 3829: Subtask get_name
[2025-05-21T16:21:35.482+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2025-04-22T00:00:00+00:00 [running]> on host 2032228d86df
[2025-05-21T16:21:35.791+0000] {taskinstance.py:3157} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-04-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-22T00:00:00+00:00'
[2025-05-21T16:21:35.793+0000] {taskinstance.py:740} INFO - ::endgroup::
[2025-05-21T16:21:35.799+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Jerry', 'last_name': 'Fridman'}
[2025-05-21T16:21:35.955+0000] {taskinstance.py:349} INFO - ::group::Post task execution logs
[2025-05-21T16:21:35.956+0000] {taskinstance.py:361} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_name, run_id=scheduled__2025-04-22T00:00:00+00:00, execution_date=20250422T000000, start_date=20250521T162135, end_date=20250521T162135
[2025-05-21T16:21:36.172+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-21T16:21:36.303+0000] {taskinstance.py:3924} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-21T16:21:36.317+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
